api/proto/common/common.proto

```diff
@@ -10,11 +13,20 @@ message G1Commitment {
 }
 
 // BlobCommitment represents commitment of a specific blob, containing its
-// KZG commitment, degree proof, the actual degree, and data length in number of symbols.
+// KZG commitment, degree proof, the actual degree, and data length in number of symbols (field elements).
+// It deserializes into https://github.com/Layr-Labs/eigenda/blob/ce89dab18d2f8f55004002e17dd3a18529277845/encoding/data.go#L27
+//
+// See https://github.com/Layr-Labs/eigenda/blob/master/docs/spec/attestation/encoding.md#validation-via-kzg
+// to understand how this commitment is used to validate the blob.
 message BlobCommitment {
+  // Concatenation of the x and y coordinates of `common.G1Commitment`.
   bytes commitment = 1;
+  // Serialization of the G2Commitment to the blob length.
   bytes length_commitment = 2;
+  // Serialization of the G2Affine element representing the proof of the blob length.
   bytes length_proof = 3;
+  // The length of the blob in symbols (field elements).
+  // TODO: is this length always a power of 2? Are there any other characteristics that we should list? etc.
   uint32 length = 4;
 }
```

Comment:
Does anyone know for this one?

Reply:
Thanks, rebased against master which has the latest changes from cody's PR, where this TODO is solved.